if __name__ == "__main__":
    try:
        pass
    except Exception as e:
        print('Error occured:', e, sep='\n')